### ISSUE TITLE
Failure in TestMetaData

### ISSUE THREAD

**Issue by dstango** [sproket/Persism#15]
Running TestMetaData results in a failure:

```
-------------------------------------------------------------------------------
Test set: net.sf.persism.TestMetaData
-------------------------------------------------------------------------------
Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.02 s <<< FAILURE! - in net.sf.persism.TestMetaData
net.sf.persism.TestMetaData.testGuessing  Time elapsed: 0.009 s  <<< FAILURE!
junit.framework.ComparisonFailure: Message s/b 'Could not determine a table for type: net.sf.persism.TestDerby Guesses were: [TestDerby, TestDerbies, Test Derby, Test_Derby, Test Derbies, Test_Derbies] and we found multiple matching tables: [TEST_DERBY, TESTDERBY]' expected:<...rbies, Test_Derbies][ and we found multiple matching tables: [TEST_DERBY, TESTDERBY]]> but was:<...rbies, Test_Derbies][]>
	at net.sf.persism.TestMetaData.testGuessing(TestMetaData.java:70)
```

The test seems to assume the existence of two tables in the Derby-database: TEST_DERBY and TESTDERBY.
Maybe you somehow created them a long time ago manually. 

```
        try {
            session.insert(new TestDerby());
        } catch (PersismException e) {
            failed = true;
            assertEquals("Message s/b 'Could not determine a table for type: net.sf.persism.TestDerby Guesses were: [TestDerby, TestDerbies, Test Derby, Test_Derby, Test Derbies, Test_Derbies] and we found multiple matching tables: [TEST_DERBY, TESTDERBY]'",
                    "Could not determine a table for type: net.sf.persism.TestDerby Guesses were: [TestDerby, TestDerbies, Test Derby, Test_Derby, Test Derbies, Test_Derbies] and we found multiple matching tables: [TEST_DERBY, TESTDERBY]",
                    e.getMessage());
        }
```

Easiest solution seems to be to create the required tables in the test:
```
diff --git a/test/net/sf/persism/TestMetaData.java b/test/net/sf/persism/TestMetaData.java
index 105fc69..cbb9fe7 100644
--- a/test/net/sf/persism/TestMetaData.java
+++ b/test/net/sf/persism/TestMetaData.java
@@ -41,0 +42 @@
+        createTables();
@@ -43,0 +45,8 @@
+    private void createTables() {
+        if (!UtilsForTests.isTableInDatabase("TEST_DERBY", con)) {
+            session.execute("CREATE TABLE TEST_DERBY (X INTEGER)");
+        }
+        if (!UtilsForTests.isTableInDatabase("TESTDERBY", con)) {
+            session.execute("CREATE TABLE TESTDERBY (X INTEGER)");
+        }
+    }
```

This makes the test pass for me.

**Comment by sproket** [comment]
Weird. I don't get a failure.  I'm assuming they don't exist.  The test is to see the name guesses for tables. When you have a class I use set of rules to see if I can find a matching table name. 

It might be somehow failing on the error message comparison. But for me it's working.

**Comment by dstango** [comment]
The expected value of the error is this:

`"Could not determine a table for type: net.sf.persism.TestDerby Guesses were: [TestDerby, TestDerbies, Test Derby, Test_Derby, Test Derbies, Test_Derbies] and we found multiple matching tables: [TEST_DERBY, TESTDERBY]"`

How would you know multiple matching tables without consulting the database?

I'd guess if you delete your derby-database in ~/pinfderby so it gets recreated you'll run into the error ....

**Comment by sproket** [comment]
I come up with possible names depending on the class name. Then I check the DB.

Ok thanks. I'll check it.

**Comment by sproket** [comment]
fixed in 1.1.0
